static/scripts/theme-blog.js

```diff
@@ -16,12 +26,8 @@ function fetchContent() {
 		timeout: 8000,
 	})
 		.done((result) => {
-			$(`.${ghostTheme}-blog .title`).text(result.pages[0].title);
-			$(`.${ghostTheme}-blog .content`).html(
-				stripHtml(result.pages[0].html, {
-					onlyStripTags: ['script', 'style'],
-				}),
-			);
+			finalHtml = result.pages[0].html;
```

Comment:
why do you need this assignment here and why do you need a global variable here. Probably it is better to do it direct in the function. And finalHtml is not a descriptive name.

Reply:
done ;)